examples/shadow-dom/main.tsx

```diff
@@ -0,0 +1,19 @@
+import ReactDOM from "react-dom/client";
+import { Button } from "@navikt/ds-react";
+import styles from "@navikt/ds-css/dist/index.css?inline";
```

Comment:
Dette er for at Vite ikke skal legge styles i header (https://vitejs.dev/guide/features.html#disabling-css-injection-into-the-page)